artemis/src/main/java/com/artemis/managers/TagManager.java

```diff
@@ -17,21 +20,53 @@
  * 
  * @author Arni Arent
  */
-public class TagManager extends Manager {
+public class TagManager extends BaseSystem {
```

Comment:
I was against it in the past, but the more I think about it, the more sense it makes